amanuensis/resources/project/__init__.py

```diff
@@ -144,7 +144,10 @@ def update(project_id, approved_url, filter_set_ids):
         return update_project(session, project_id, approved_url)
 
 
-def upload_file(bucket, key, project_id, expires=None):
+def upload_file(key, project_id, expires=None):
+
+    bucket = config["S3_BUCKETS"]["PROJECT_DATA_BUCKET"]
```

Comment:
this is the correct one for the data request: https://github.com/chicagopcdc/amanuensis/blob/131f1962987ddffd4af2e8409aa934c4d824d0f3/amanuensis/config-default.yaml#L153-L154